Fix the Enable Windows version unit test

TrustHenry opened the pull request:
This runs a unit test of the Windows version.
In the Windows version, due to problems with the parallelism test,
the test is performed with single threads.
And Fix the can't allocate of push_back

Related to #820

AndrejMitrovic commented:
I recommended opening the PR with this because the "cannot allocate" issues Henry is seeing might be because of running out of RAM when running the VM. Let's see..

Geod24 commented on source/agora/test/Base.d:
```diff
@@ -185,8 +187,15 @@ private UnitTestResult customModuleUnitTester ()
 
-    foreach (mod; parallel(parallel_tests))
-        runTest(mod);
+    version (Windows)
+    {
+        // TODO
+    }
+    else
+    {
+        foreach (mod; parallel(parallel_tests))
+            runTest(mod);
+    }
```
No! The code I gave you yesterday would run the test single-threaded. This one just excludes the test, which is absolutely not what we want.

TrustHenry replied:
Yes, I proceeded with the missing test. 
In the Windows version, modify to test only single threads.

TrustHenry committed:
Correct incorrt branching in windows push_back

TrustHenry committed:
Fix the Enable Windows version unittest

This runs a unit test of the Windows version.
In the Windows version, due to problems with the parallelism test, the test is performed with single threads.

TrustHenry commented:
It feels good. It's all green. without (ci/dockercloud)
All of Fiber's stackSize=512KB were successful in CI Tests on Ubuntu, macOS, and Windows-2019 OS.

TrustHenry commented on source/agora/test/Base.d:
```diff
@@ -183,13 +188,22 @@ private UnitTestResult customModuleUnitTester ()
     // Run single-threaded tests
     foreach (mod; single_threaded)
+    {
+        writefln("single_thread_tests/%s", mod.name);
```
I added a print for the test. I'm going to remove it.

TrustHenry commented:
What do we think about changing the `Fiber` stack size of the `localrest`?

TrustHenry commented:
Ready for review. All Green!
